### PR TITLE
OSSM-8241: Update OSSM release notes

### PR DESCRIPTION
**Pull request by rh-tokeefe**
Version(s): OCP 4.14+

Issue: https://issues.redhat.com/browse/OSSM-8241

Link to docs preview:
https://83251--ocpdocs-pr.netlify.app/openshift-enterprise/latest/service_mesh/v2x/servicemesh-release-notes.html#gateway-api-ga-cluster-wide-deployments-ossm-2-6-0_ossm-release-notes

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Review by pbajjuri20 — APPROVED** [review]
LGTM..!

File: modules/ossm-release-2-6-0.adoc [hunk 1]
```diff
@@ -114,12 +114,31 @@ When updating existing instances of the `ServiceMeshControlPlane` resource to {S
 
 {SMProductName} 2.6 is the last release that includes support for {JaegerName} and {es-op}. Both {JaegerShortName} and {es-op} will be removed in the next release. If you are currently using {JaegerShortName} and {es-op}, you need to switch to {TempoName} and {OTELName}.
 
-//Gateway API Update for 2.6 OSSM-5854
+
```
Comment (errantepiphany) [inline modules/ossm-release-2-6-0.adoc:117]: Just curious why so many empty lines?

Comment (rh-tokeefe) [reply modules/ossm-release-2-6-0.adoc]: updated PR to remove.

**Review by longmuir — APPROVED** [review]
LGTM - thank you!

**Comment by rh-tokeefe** [comment]
The Doc PM commented in [OSSM-8241](https://issues.redhat.com/browse/OSSM-8241)  and said the CM process is not required for this change.

File: modules/ossm-release-2-6-0.adoc [hunk 2]
```diff
@@ -114,12 +114,11 @@ When updating existing instances of the `ServiceMeshControlPlane` resource to {S
...
+//Gateway API Update for 2.6 OSSM-5854 subsequently revised by OSSM-8241
 //Kubernetes Gateway API and {product-title} Gateway API are the same. It is referenced as {product-title} Gateway API in 2.5 and as {product-title} Gateway API here https://docs.openshift.com/container-platform/4.15/nodes/clusters/nodes-cluster-enabling-features.html so to be consistent, it is also referenced as {product-title} Gateway API for 2.6.
 [id="gateway-api-ga-cluster-wide-deployments-ossm-2-6-0_{context}"]
-== {product-title} Gateway API generally available for cluster-wide deployments
-//Jacek. Approved 07/11/2024
-This release introduces the General Availability of {product-title} Gateway API, also known as the Kubernetes Gateway API, which is enabled by default only for cluster-wide deployments.
+== Gateway API use is generally available for OpenShift {SMProductShortName} cluster-wide deployments
+This release introduces the General Availability for using the Kubernetes Gateway API version 1.0.0 with OpenShift {SMProductShortName} 2.6. This API use is limited to OpenShift {SMProductShortName} and the Gateway API custom resource definitions (CRDs) are not supported.
```
Comment (jeana-redhat) [inline modules/ossm-release-2-6-0.adoc:121]: Should `OpenShift {SMProductShortName}` ("OpenShift Service Mesh") actually be `{SMProductName}` ("Red Hat OpenShift Service Mesh")? (generally "OpenShift" should not appear in plain text)

```suggestion
== Gateway API use is generally available for {SMProductName} cluster-wide deployments
This release introduces the General Availability for using the Kubernetes Gateway API version 1.0.0 with {SMProductName} 2.6. This API use is limited to {SMProductShortName} and the Gateway API custom resource definitions (CRDs) are not supported.
```

Comment (jeana-redhat) [inline modules/ossm-release-2-6-0.adoc:121]: Also suggest breaking that last sentence up for clarity.
```suggestion
== Gateway API use is generally available for OpenShift {SMProductShortName} cluster-wide deployments
This release introduces the General Availability for using the Kubernetes Gateway API version 1.0.0 with OpenShift {SMProductShortName} 2.6. This API use is limited to OpenShift {SMProductShortName}. The Gateway API custom resource definitions (CRDs) are not supported.
```

**Review by jeana-redhat — COMMENTED** [review]
Just a couple things to look at

/remove-label peer-review-in-progress
/remove-label peer-review-needed
/label peer-review-done

File: modules/ossm-release-2-6-0.adoc (hunk 2 above)
Comment (rh-tokeefe) [reply modules/ossm-release-2-6-0.adoc:121]: done

Comment (rh-tokeefe) [reply modules/ossm-release-2-6-0.adoc:121]: done

**Review by skopacz1 — APPROVED** [review]
Merge review LGTM

**Pull request merged**